test/scriptFlywheel/unit/borrowRatePerBlockTests.spec.ts

```diff
@@ -29,6 +29,8 @@ contract("SavingAccount.liquidate", async (accounts) => {
     const dummy = accounts[9];
     const eighteenPrecision = new BN(10).pow(new BN(18));
     const sixPrecision = new BN(10).pow(new BN(6));
+    const eightPrecision = new BN(10).pow(new BN(8));
+    const OKEX_BPY = new BN(10512000);
```

Comment:
variable name should be `ETH_BPY`. I would recommend to use only BLOCKS_PER_YEAR.

Reply:
Sure